Jenkinsfile

```diff
@@ -5,6 +5,7 @@ pipeline {
 
   options {
     timestamps()
+    disableConcurrentBuilds()
```

Comment:
does this mean that all the steps will run asynchronously?  How much time does it add to each build?

Reply:
It means i cant run on the same branch two jobs in parallel. Why? since we are using same oc environment. It cause to some of the failures.

Reply:
on the same git branch? So if we run 2 builds on the same branch they will wait in a queue?
I'd like to get @hughsaunders 's take on this.

Reply:
if not.. you will run on OC environment more than one job and we can see gcloud crashed again.

Reply:
I see. Please add a comment explaining why this line is important.

Reply:
👍